### PR TITLE
READY - Bake version numbers into all binaries.

### PR DESCRIPTION
**Pull request by philwinder**
Part of #248.
Adds `--version` parameter to `fluxsvc` and `fluxd` binaries.

### CODE REVIEW

File: cmd/fluxd/version_test.go [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+	"testing"
+)
+
+func TestVersion(t *testing.T) {
+	RunVersioned(t, "", "undefined")
+	RunVersioned(t, "1.2.3", "1.2.3")
+}
+
+func RunVersioned(t *testing.T, version string, expected string) {
+	version = version
```
Comment (rade) [inline cmd/fluxd/version_test.go]: huh?

File: cmd/fluxd/version_test.go [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+	"strings"
+	"testing"
+)
+
+func TestVersion(t *testing.T) {
+	RunVersioned(t, "", "undefined")
+	RunVersioned(t, "1.2.3", "1.2.3")
+}
+
+func RunVersioned(t *testing.T, version string, expected string) {
```
Comment (rade) [inline cmd/fluxd/version_test.go]: `expected` isn't used anywhere.

File: cmd/fluxd/version_test.go (hunk 1 above)
Comment (philwinder) [inline cmd/fluxd/version_test.go]: the global main.version = tempversion. Will rename.

File: cmd/fluxd/version_test.go (hunk 2 above)
Comment (philwinder) [inline cmd/fluxd/version_test.go]: Indeed.

**Pull request merged**